src/pages/SignUp/SignUp.jsx

```diff
@@ -1,7 +1,157 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import './SignUp.scss';
 
 const SignUp = () => {
-  return <div></div>;
+  const [userInfo, setUserInfo] = useState({
+    memberId: '',
+    memberPassword: '',
+    memberName: '',
+    memberPhonenumber: '',
+    memberEmail: '',
+    memberBirthday: '',
+  });
+  const [memberGender, setMemberGender] = useState('');
+
+  const {
+    memberId,
+    memberPassword,
+    memberName,
+    memberPhonenumber,
+    memberEmail,
+    memberBirthday,
+  } = userInfo;
+
+  const onClickSignup = e => {
+    e.preventDefault();
+
+    fetch('http://10.58.52.244:3000/member/signup', {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify({
+        memberId,
+        memberEmail,
+        memberPassword,
+        memberName,
+        memberPhonenumber,
+        memberBirthday,
+        memberGender,
+      }),
```

Comment:
백엔드에 보내야 하는 key와 state 객체의 key를 맞추었기 때문에, state 자체를 전달해도 될 것 같습니다.
```suggestion
      body: JSON.stringify(userInfo),
```